02-intermediate-ruby/exercises/sample_code/order.rb

```diff
@@ -0,0 +1,57 @@
+
+class OrderItem
+  attr_reader :name, :amount, :unit_price, :is_alcohol
+
+  def initialize(name, amount, unit_price, is_alcohol = false)
+    @name = name
+    @amount = amount
+    @unit_price = unit_price
+    @is_alcohol = is_alcohol
+  end
+
+  def total_sale
+    return unit_price * amount
+  end
+end
+
+class Order
+  attr_reader :order_items
+
+  def initialize(items = [])
+    @order_items = items
+  end
+
+  def add_item(item)
+    order_items << item
```

Comment:
Will this work without the @ (since the attr is the reader)?

Reply:
It will since it's an array (references)